src/private/utils.ts

```diff
@@ -30,19 +30,7 @@ export function decorate(handleDescriptor, entryArgs) {
 }
 
 class Meta {
-  @lazyInitialize
-  debounceTimeoutIds = {};
-
-  @lazyInitialize
-  throttleTimeoutIds = {};
-
-  @lazyInitialize
-  throttlePreviousTimestamps = {};
-
-  @lazyInitialize
-  throttleTrailingArgs = null;
-
-  @lazyInitialize
+  // @lazyInitialize
```

Comment:
I remember commenting out the @lazyInitialize part was intentional.   Its broken due to TypeScript's limitations on property initializers, tests from Babel fail with the @lazyInitialize in place.

As I remember removing the `throttleTrailingArgs = null;` part was part of removing the obsolete decorators and their tests.